packages/nimble-components/CONTRIBUTING.md

```diff
@@ -428,6 +428,31 @@ Nimble includes three NI-brand aligned themes (i.e. `light`, `dark`, & `color`).
 
 When creating a new component, create a `*-matrix.stories.ts` Storybook file to confirm that the component reflects the design intent across all themes and states.
 
+## Localization
+
+Nimble components may need to use some strings/labels (such as the label for a close button in a component's template, or menu items in a component-provided menu). Nimble exposes these localizable labels as design tokens, to support both localization and the ability for clients to override the strings.
```

Comment:
It would be good to update this introductory paragraph (here and in README and in storybook) to provide some more context about the normal case being that clients provide localized strings and that this is for the rarer case where Nimble is providing the string and apps need to override it. Basically make this an answer to the question "how does Nimble handle localization?"

A rough draft:
Most user-visible strings displayed by Nimble components are provided by the client application and are expected to be localized by the application if necessary. However, some strings are built in to Nimble components and are provided only in English. An application can provide localized versions of these strings by using design tokens set on label provider elements.

Reply:
Updated